CMakeLists.txt

```diff
@@ -16,13 +16,16 @@
 cmake_minimum_required(VERSION 3.14)
 get_directory_property(is-subproject PARENT_DIRECTORY)
 
+
+# C is required to find MKL automatically
 project(
   "multicharge"
-  LANGUAGES "Fortran"
+  LANGUAGES "Fortran" "C"
```

Comment:
I would prefer to only depend on a Fortran compiler here.

Reply:
I have removed this in my latest commit, but the automatic MKL detection with `cmake` will not longer work. See bottom of https://cmake.org/cmake/help/latest/module/FindLAPACK.html